### ISSUE TITLE
Launch on Startup still not working

### ISSUE THREAD

**Issue by atebeta** [timmo001/system-bridge#559]
### Description

Launch on startup is still not working for me., tried uninstalling/reinstalling as administrator. 

### What Platform / OS are you running?

Windows

### What version are you running?

2.0.1

### Anything in the logs or a references that might be useful?

```txt
error: Main - Error Opening Tray: Command failed with exit code 1073807364: C:\Users\<removed>\AppData\Local\System Bridge\system-bridge-tray.exe
```

**Comment by timmo001** [comment]
Do any more logs appear in `%appdata%\system-bridge\system-bridge.log`?

**Comment by atebeta** [comment]
No, nothing else. Here are the logs when I start it manually.

`2021-07-19 16:32:04 info: -----------------------------------------------------------------------------------------------------------------------
2021-07-19 16:32:04 info: System Bridge 2.1.0: ["C:\\Users\\<hidden>\\AppData\\Local\\System Bridge\\system-bridge.exe","C:\\snapshot\\system-bridge\\dist\\main.js"] - undefined
2021-07-19 16:32:04 info: -----------------------------------------------------------------------------------------------------------------------
2021-07-19 16:32:04 info: Main - Open Tray: C:\Users\<hidden>\AppData\Local\System Bridge\system-bridge-tray.exe
2021-07-19 16:32:05 info: Main - API started on port 9170
2021-07-19 16:32:05 info: Main - RTC broker created on path /rtc
2021-07-19 16:32:05 info: WebSocket - New client registered
2021-07-19 16:32:05 info: WebSocket - New event: open-rtc
2021-07-19 16:32:05 info: Main - Event: open-rtc
2021-07-19 16:32:05 info: Main - Launch on startup: true - enabled - C:\Users\<hidden>\AppData\Local\System Bridge\system-bridge.exe
2021-07-19 16:32:07 info: -----------------------------------------------------------------------------------------------------------------------
2021-07-19 16:32:07 info: System Bridge - Tray 2.1.0: ["C:\\Users\\<hidden>\\AppData\\Local\\System Bridge\\system-bridge-tray.exe","C:\\snapshot\\system-bridge\\tray\\dist\\tray\\src\\tray.js"] - undefined
2021-07-19 16:32:07 info: -----------------------------------------------------------------------------------------------------------------------
2021-07-19 16:32:07 info: Tray - Create Tray
2021-07-19 16:32:10 info: Tray - Started`

**Comment by FaserF** [comment]
I thought it wasn't working too, but a restart between setting the settings worked for me.

1. Install the Program
2. Set Auto Startup setting
3. Reboot
4. Check Auto Startup Setting again and check in the task manager if there is a nodeJS entry in the autostartup tab
5. if there is no nodeJS entry disable autostartup and enable it again
6. Reboot

**Comment by atebeta** [comment]
I tried the following, but still no auto start.

1. Disable autostart (i already had the nodeJS entry in the startup tab)
2. Reboot
3. Enbable autostart (nodeJS entry still in startup tab)
4. Reboot

**Comment by timmo001** [comment]
As soon as you check that setting, the settings should update. There in the logs it says enabled. Check your task manager startup tab. There should be an entry that looks like this:

![image](https://user-images.githubusercontent.com/28114703/126361645-66196717-dd39-4469-ac79-91bdfdf9fe09.png)

The name is misleading, due to the way node.js is packaged (essentially you are running node.exe with an embedded filesystem)

You can check the location of the entry in startup by right-clicking and opening file location:

![image](https://user-images.githubusercontent.com/28114703/126361814-5e5bb348-d94e-4bea-8939-fb532d4a33f3.png)

**Comment by atebeta** [comment]
I have this entry in my startup tab, but its still not working. I tried uninstalling, deleting the System Bridge folder from my %appdata% folder and reinstalling. Is there a way to do a completely fresh install? I noticed all the setting are still intact when I reinstalled.

**Comment by timmo001** [comment]
You can uninstall, and delete the `%appdata%\system-bridge` if you like. On reinstall those files will be recreated as new. You should however uncheck the startup option first before installing so you don't leave links around

**Comment by atebeta** [comment]
I tried unchecking the startup option and reinstalling, but still cannot get launch on startup to work. If the startup option is unchecked when uninstalling it removes the `%appdata%\system-bridge ` automatically, but there must still be some files left behind because the logs and settings are still saved. Is there no way to do a fresh install?

**Comment by timmo001** [comment]
This will be `C:\Users\USERNAME\AppData\Roaming\system-bridge`. there is a log and db to delete in there manually. However, this will be unlikely to fix anything as it sets the startup at the system level.

Are you using the latest version now?

**Comment by atebeta** [comment]
Yeah I'm running the latest version. Seems like the log and db were also removed when I uninstalled with the launch on startup option unselected, but its still not working when i reinstalled as you anticipated. I can see the NodJS entry being created in the startup tab of the task switcher when I enable the option, but it still doesn't seem to do anything. Not sure what else to do now other than start digging into the Windows logs, but at the moment I'm ok with just having it start via the Task Scheduler.
